src/cmd-push-container-manifest

```diff
@@ -37,6 +38,16 @@ def main():
         #   - Store the path to the container image in the container_images list
         images = []
         buildmetas = dict()
+        upload = False
+        check_digest = ""
+
+        # Checks if we already have it in the remote
+        try:
+            cmd = ["skopeo", "inspect","--raw", f"docker://{args.repo}:{args.tags[0]}"]
+            check_digest = runcmd(cmd, capture_output=True).stdout
+        except:
+            pass
```

Comment:
rather than a `try/except` here let's just pass `check=False` to `runcmd()`?

Also, maybe we should parse the json and pull the digests out rather than just slurping up all of the output?